sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10653,6 +10654,88 @@
               "hideLabel": false,
               "errorLabel": "Disability Information"
             },
+            {
+              "label": "HTML",
+              "labelWidth": "",
+              "labelMargin": "",
+              "tag": "p",
+              "className": "alert alert-warning fa fa-exclamation-triangle w-100",
+              "attrs": [
+                {
+                  "attr": "",
+                  "value": ""
+                }
+              ],
+              "content": "<strong> Your account status indicates you have been approved for Disability status.</strong>\n <br /> If you answer no to this question you will not be assessed or receive disability funding.",
+              "refreshOnChange": false,
+              "customClass": "banner-warning",
+              "hidden": false,
+              "modalEdit": false,
+              "key": "html8",
+              "tags": [],
+              "properties": {},
+              "conditional": {
```

Comment:
Please remove the simple condition. 

![image](https://github.com/bcgov/SIMS/assets/54600590/9c8a4399-80ac-443d-bdf6-4c75bfdb12c5)

Reply:
Thanks, this should now be updated on all the forms.